### DISCUSSION TITLE
Unpublish or deprecate a particular extension version

### DISCUSSION THREAD

**Discussion by microhobby** [microsoft/vscode-discussions#371] [Extension Development QnA] [answered]
Let me explain my problem. I wrongly published my extension as `multi-platform` package, but it only works on `linux-x64`. Now that I'm publishing packages correctly for the right platform users can still install the latest `multi-platform` version which causes problems and confusion.

Is there a way to remove or deprecate specific versions or packages of an extension?

**Comment by amit-kulkarni-in** [comment, marked as answer]
@microhobby : Please reach out to VSMarketplace@microsoft.com .
